code/game/objects/items/stacks/sheets/leather.dm

```diff
@@ -221,15 +221,42 @@ GLOBAL_LIST_INIT(sinew_recipes, list (
 		var/mob/living/silicon/robot/R = target
 		if(istype(R.module, /obj/item/robot_module/miner))
 			var/datum/armor/current_armor = R.armor
-			if(current_armor.getRating(MELEE) < 60)
-				R.armor = current_armor.setRating(melee_value = min(current_armor.getRating(MELEE) + 10, 60))
+			if(current_armor.getRating(MELEE) < 75)
+				R.armor = current_armor.setRating(melee_value = min(current_armor.getRating(MELEE) + 15, 75))
 				to_chat(user, "<span class='info'>You strengthen [target], improving its resistance against melee attacks.</span>")
 				use(1)
```

Comment:
You should also check earlier if you can actually use a single plate from the stack, like sirryan's review

Reply:
Dont see need for that, all works perfectly, from stack uses 1 plate, not all or somehow different